components/project/Submissions.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import ReactPlayer from 'react-player';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import TextHeader from '../TextHeader';
+import { ISubmission } from '../../models/Submission';
+
+interface ISub {
+	data: ISubmission,
+	index: number
+}
+
+const Submission = ({ data, index }: ISub) => (
```

Comment:
```suggestion
interface ISubmissionProps {
	data: ISubmission,
	index: number
}

const Submission = ({ data, index }: ISubmissionProps) => (
```
For naming consistency